tools/base/test/base.test.ts

```diff
@@ -10,6 +10,7 @@ governing permissions and limitations under the License.
 */
 import { SpectrumElement } from '@spectrum-web-components/base';
 import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+import { version } from '../src/version';
```

Comment:
File extension.

Reply:
@Westbrook After adding the file extensions I see that all tests are failing reason being the version.js is being created/updated using genversion & it's excluded from build so version.dev.js is not created & all the test's fail. 
Is it okay to roll back to removing extension from version as only version.js file is being created ?

Reply:
File extensions are required. Checkout my other note re `*.dev.js` files.